styles.js

```diff
@@ -23,11 +24,13 @@ ul {
 }
 
   :root {
-// Colors
---color-green: #426B1F;
+// Colors general 
+--color-green: #426B1F; 
 --color-beige: #FAFAF5;
 --color-black: #010101;
 --color-grey: #E6E6E6;
 --color-lightGreen: #53A00A;
+--color-blackGreen: #2B2E28;
+--color-darkGreen: #3E4438;
```

Comment:
why you keep that kind of color declarations? Wouldnt it be better to have one theming solution?